fotoapparat/src/main/java/io/fotoapparat/view/CameraView.java

```diff
@@ -52,8 +57,21 @@ public CameraView(@NonNull Context context,
 
     private void init() {
         rendererView = new TextureRendererView(getContext());
-
         addView(rendererView);
+
+        focusMarkerLayout = new FocusMarkerLayout(getContext());
+        focusMarkerLayout.setOnTouchListener(new OnTouchListener() {
+            @Override
+            public boolean onTouch(View v, MotionEvent motionEvent) {
+                if (motionEvent.getAction() == MotionEvent.ACTION_DOWN && isTapToFocusEnabled) {
+                    focusMarkerLayout.showAt(motionEvent.getX(), motionEvent.getY());
+                    rendererView.dispatchTouchEvent(motionEvent);
```

Comment:
There is no need for this method if you will always return `false` from this callback.